kedro_mlflow/framework/hooks/pipeline_hook.py

```diff
@@ -62,9 +66,13 @@ def before_pipeline_run(
             pipeline: The ``Pipeline`` that will be run.
             catalog: The ``DataCatalog`` to be used during the run.
         """
-        mlflow_conf = get_mlflow_config(
-            project_path=run_params["project_path"], env=run_params["env"]
+        context = load_context(
```

Comment:
context -> self.context (in case we need it elsewhere, for now it should have no impact)

Reply:
Well spoted, it's a `self.context`